docker/docker-compose.1804.54.yaml

```diff
@@ -3,14 +3,16 @@ version: "3"
 services:
 
   runtime-setup:
-    image: swift-distributed-tracing-baggage:18.04-5.4
+    image: swift-distributed-tracing-baggage:20.04-5.4
     build:
       args:
-        ubuntu_version: "bionic"
+        ubuntu_version: "focal"
```

Comment:
the file is called 18.04 so probably want a new file instead of modifying this one

Reply:
Thanks!